superset/dataframe.py

```diff
@@ -19,6 +22,21 @@
 
 
 class SupersetDataFrame(object):
+    # Mapping numpy dtype.char to generic database types
+    type_map = {
+        'b': 'BOOL',  # boolean
+        'i': 'INT',  # (signed) integer
+        'u': 'INT',  # unsigned integer
+        'f': 'FLOAT',  # floating-point
+        'c': 'FLOAT',  # complex-floating point
+        'm': None,  # timedelta
```

Comment:
I wanted to cover the entire scope of numpy types to be explicit, but it really doesn't map to a classic database type. We could handle / convert those in the future but most likely database drivers will never return that type (though they are supported in numpy...)